userspace/libsinsp/socket_handler.h

```diff
@@ -388,16 +392,17 @@ class socket_data_handler
 				//			 "\n\n" + data + "\n\n", sinsp_logger::SEV_TRACE);
 			}
 
-                        // To prevent reads from entirely stalling (like in gigantic k8s
-                        // environments), give up after reading 30mb.
+			// To prevent reads from entirely stalling (like in gigantic k8s
+			// environments), give up after reading 30mb.
```

Comment:
This comment is no longer entirely accurate

Reply:
Thank you. This should be addressed in efa93793dfa05e1ae26562ec57847bf7ec700525.